cdc/sink/mysql/mysql_syncpoint_store.go

```diff
@@ -198,21 +216,35 @@ func (s *mysqlSyncpointStore) SinkSyncpoint(ctx context.Context,
 		}
 		return cerror.WrapError(cerror.ErrMySQLTxnError, err)
 	}
-	query := "insert ignore into " + schemaName + "." + syncpointTableName +
-		"(cf, primary_ts, secondary_ts) VALUES (?,?,?)"
-	_, err = tx.Exec(query, id.Namespace+"_"+id.ID, checkpointTs, secondaryTs)
+	query := "insert ignore into " + schemaName + "." + syncPointTableName +
+		"(ticdc_cluster_id, cf, primary_ts, secondary_ts) VALUES (?,?,?,?)"
+	_, err = tx.Exec(query, s.clusterID, id.Namespace+"_"+id.ID, checkpointTs, secondaryTs)
 	if err != nil {
 		err2 := tx.Rollback()
 		if err2 != nil {
 			log.Error("failed to write syncpoint table", zap.Error(cerror.WrapError(cerror.ErrMySQLTxnError, err2)))
 		}
 		return cerror.WrapError(cerror.ErrMySQLTxnError, err)
 	}
+
+	query = fmt.Sprintf(
+		"DELETE IGNORE FROM "+schemaName+"."+syncPointTableName+"WHERE cf like '%s' and creat_at < (NOW() - INTERVAL %.2f SECOND)",
+		id.Namespace+"_"+id.ID,
+		s.syncPointRetention.Seconds())
+	_, err = tx.Exec(query)
+	if err != nil {
+		err2 := tx.Rollback()
```

Comment:
if failed to remove the old sync-point recorder , writing a log and ignoring this error  is ok

Reply:
got it